src/knot_cloud.c

```diff
@@ -469,11 +421,13 @@ int knot_cloud_unregister_device(const char *id)
 	 * Expiration
 	 *	2000 ms
 	 */
-	result = mq_publish_direct_message(MQ_EXCHANGE_DEVICE,
-					   MQ_CMD_DEVICE_UNREGISTER,
-					   headers, 1,
-					   MQ_MSG_EXPIRATION_TIME_MS,
-					   json_str);
+	mq_message_data_t mq_message = {
+				 MQ_MESSAGE_TYPE_DIRECT, MQ_EXCHANGE_DEVICE,
+				 MQ_CMD_DEVICE_UNREGISTER, MQ_MSG_EXPIRATION_TIME_MS, json_str,
```

Comment:
Could you modify this part of code to fit in 80 characters?

Reply:
I think the problem is related to github, in my text editor this column length is 78